wazo_dird/plugins/phonebook/http.py

```diff
@@ -34,70 +38,73 @@ def _make_error(reason, status_code):
     )
 
 
-class _Resource(LegacyAuthResource):
-    def __init__(self, phonebook_service, auth_client):
-        self.phonebook_service = phonebook_service
-        self._auth_client = auth_client
+class PhonebookServiceProtocol(Protocol):
```

Comment:
Added explicit interface definition for PhonebookService here, in order to maintain separation with other plugins.
This could be extracted to a `interfaces` module in this plugin, or into a separate utility namespace for `phonebook` interfaces(!= implementations) shared with other phonebook plugins.

Reply:
(removed this abstract interface following discussion with @sduthil , importing from phonebook_service plguin directly)